fedn/fedn/utils/helpers.py

```diff
@@ -49,7 +49,7 @@ def get_helper(helper_type):
     if helper_type == 'numpymodel':
         from fedn.utils.numpymodel import NumpyHelper
         return NumpyHelper()
-    elif helper_type == 'keras_weights':
+    elif helper_type == 'keras':
         from fedn.utils.kerasweights import KerasWeightsHelper
```

Comment:
Should be kerashelper and KerasHelper here (updated this in 0.2). You might not have the latest upstream code?

Reply:
Now it should work after pull